src/test/java/seedu/address/testutil/TypicalReviews.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.testutil;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import seedu.address.model.eatery.Review;
+
+/**
+ * Contains a list of typical reviews used for testing.
+ */
+public class TypicalReviews {
+
+    private static DateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy");
+    private static Date date;
+    static {
+        try {
+            date = dateFormat.parse("22/11/2019");
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
isn't this supposed to be public and not private?

Reply:
I'm assuming `dateFormat` and `date` are not accessed from outside this class, so it's ok for them to be private. Since the static `Review` objects reference them (and they are static), `dateFormat` and `date` need to be static as well. So all this looks fine 👍